ibm/mas_devops/roles/minio/README.md

```diff
@@ -0,0 +1,53 @@
+# Minio
+=====
+
+This role provides support to install and configure Minio Storage
+
+* Install Minio storage
+
+Role Variables
+--------------
+
+### minio_namespace
+Action to be performed by minio role. Valid values are `string`.
```

Comment:
Descriptions need updating in all of these. `Action to be performed by minio role.` is not a valid description